components/item-act.js

```diff
@@ -35,50 +37,110 @@ const addCustomTip = (amount) => {
   localStorage.setItem('custom-tips', JSON.stringify(customTips))
 }
 
-export default function ItemAct ({ onClose, itemId, act, strike }) {
+export default function ItemAct ({ onClose, item, act, strike }) {
   const inputRef = useRef(null)
   const me = useMe()
   const [oValue, setOValue] = useState()
+  const client = useApolloClient()
 
   useEffect(() => {
     inputRef.current?.focus()
-  }, [onClose, itemId])
+  }, [onClose, item.id])
 
-  return (
-    <Form
-      initial={{
-        amount: me?.tipDefault,
-        default: false
-      }}
-      schema={amountSchema}
-      onSubmit={async ({ amount }) => {
-        await act({
-          variables: {
-            id: itemId,
-            sats: Number(amount)
+  const updateCache = (itemSats) => {
+    client.cache.modify({
+      id: `Item:${item.id}`,
+      fields: {
+        sats (existingSats = 0) {
+          return existingSats + itemSats
+        },
+        meAnonSats (anonSats = 0) {
+          return anonSats + itemSats
```

Comment:
for some reason, this function is not called thus `meAnonSats` is not updated :thinking: